charts/osm/templates/fluentbit-configmap.yaml

```diff
@@ -21,8 +21,13 @@ data:
       match      *
       regex       message /"level":"error"/
     [OUTPUT]
-      Name    stdout
+      Name    {{ .Values.OpenServiceMesh.fluentbitOutput }}
```

Comment:
Maybe this whole section under `[OUTPUT]` should be parameterized as each output plugin will have its own configurations. 
WDYT @SanyaKochhar

Reply:
On second thought, it's good to have the ability to set these fields individually.